### PR TITLE
❄️ igraph 0.10.13

### PR DESCRIPTION
**Pull request by SC426**
### Channels
- Snowflake

### Links

- [PKG-4629](https://anaconda.atlassian.net/browse/PKG-4629)

[PKG-4629]: https://anaconda.atlassian.net/browse/PKG-4629?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/patches/0001_remove_safelocale.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
```
Comment (danpetry) [inline recipe/patches/0001_remove_safelocale.patch:10]: header on the patch to explain would be good

File: recipe/meta.yaml [hunk 2]
```diff
@@ -44,7 +46,7 @@ requirements:
     - mkl 2023.1.0  # [win]
   run:
     - glpk  # [not (linux and s390x)]
-    - libxml2
+    - {{ pin_compatible('libxml2') }}
```
Comment (danpetry) [inline recipe/meta.yaml]: libxml2 has a run_exports already, we don't want to override it here

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://github.com/igraph/igraph/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
-  sha256: aa5700b58c5f1e1de1f4637ab14df15c6b20e25e51d0f5a260921818e8f02afc
+  sha256: c6dc44324f61f52c098bedb81f6a602365d39d692d5068ca4fc3734b2a15e64c
+  patches:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  patches:  # [osx]
```
For consistency

File: recipe/patches/0001_remove_safelocale.patch (hunk 1 above)
Comment (SC426) [reply recipe/patches/0001_remove_safelocale.patch:10]: Done!

File: recipe/meta.yaml (hunk 3 above)
Comment (SC426) [reply recipe/meta.yaml]: Done! Thanks!

**Review by skupr-anaconda — APPROVED** [review]

**Review by danpetry — APPROVED** [review]

**Pull request merged**